src/Painter.js

```diff
@@ -477,8 +477,8 @@ Painter.prototype = {
             ctx.restore();
         }
 
-        if (env.wxa) {
-            // Flush for weixin application
+        if (env.miniApp) {
```

Comment:
wxa should not be removed. Because env.wxa has been exported to developers.
and Baidu swan app is the different environment from wechat mini app.
If we intend to support Baidu swan app, probably we can add a new mark on env, 
like 'env.swan' or 'env.baiduSwan'  or 'env.bdswan' or something.

What's your opinion? @Ovilia